linkerd/app/outbound/src/http/concrete.rs

```diff
@@ -9,20 +9,30 @@ use linkerd_app_core::{
     },
     svc, Error, Infallible,
 };
+use tracing::info_span;
 
 impl<N> Outbound<N> {
+    /// Builds a [`svc::NewService`] stack that builds buffered HTTP load
+    /// balancer services for [`Concrete`] targets.
+    ///
+    /// When a balancer has no available inner services, it goes into
+    /// 'failfast'. While in failfast, buffered requests are failed and the
+    /// service becomes unavailable so callers may choose alternate concrete
+    /// services.
+    //
+    // TODO(ver) make the outer target type generic/parameterized.
     pub fn push_http_concrete<NSvc, R>(
         self,
         resolve: R,
     ) -> Outbound<
         svc::ArcNewService<
             Concrete,
             impl svc::Service<
-                http::Request<http::BoxBody>,
-                Response = http::Response<http::BoxBody>,
-                Error = Error,
-                Future = impl Send,
-            >,
+                    http::Request<http::BoxBody>,
+                    Response = http::Response<http::BoxBody>,
+                    Error = Error,
+                    Future = impl Send,
+                > + Clone,
```

Comment:
tiny nit: is this indentation rustfmt-approved? it _seems_ off to me

Reply:
Yeah, that's all rustfmt's doing.